packages/components/psammead-most-read/src/testHelpers/index.jsx

```diff
@@ -18,6 +18,16 @@ const lastUpdated = ({ script, service }) => (
   </Timestamp>
 );
 
+export const getServiceVariant = ({ service, variant = '' }) => {
+  // non variant services have a default value
+  const variantOverride =
+    variant === 'default'
+      ? ''
+      : variant.charAt(0).toUpperCase() + variant.substring(1);
+
+  return service + variantOverride;
+};
+
```

Comment:
I think this looks cleaner:
```
if (variant !== 'default') {
  const variantOverride = variant.charAt(0).toUpperCase() + variant.substring(1);
  return service + variantOverride;
}

return service;
```

Reply:
Agree with this, adding this now in Khoa's absence.